weave/tests/test_uploader.py

```diff
@@ -1453,3 +1462,69 @@ def test_upload_from_s3fs(test_basket):
         s3.rm(minio_path,recursive=True)
     if local_fs.exists(pantry_2.index.db_path):
         local_fs.rm(pantry_2.index.db_path)
+
+# Skip tests if pymongo is not installed.
+@pytest.mark.skipif(
+    "pymongo" not in sys.modules or not os.environ.get("MONGODB_HOST", False),
+    reason="Pymongo required for this test",
+)
+def test_upload_basket_mongo(test_basket):
+    """
+    Testing pantry.upload_basket(), expected
+    to update the collections in mongodb
+    """
+    ch10_path = str(resources.files(test_data) /
+                    "652200104150842.ch10")
+    fs = test_basket.file_system
+    pantry_path = test_basket.pantry_path
+
+    pantry = Pantry(IndexPandas,pantry_path=pantry_path,file_system=fs)
+
+    uuid = pantry.upload_basket(
+                                upload_items=[{'path':ch10_path,
+                                'stub':False}], basket_type="test-1",
+                                metadata = {'Data Type':'ch10'}
+                               ).values.tolist()[0][0]
+
+    collections = ("test_supplement", "test_metadata", "test_manifest")
+    mongo_client = get_mongo_db()
+    mongo_db = mongo_client["test_mongo_db"]
+    query = {'uuid': uuid}
+
+    for e in collections:
+        assert uuid == mongo_db[e].find_one(query,{'_id':0,'uuid':1})['uuid']
+        mongo_db[e].delete_one(query)
+
+# Skip tests if pymongo is not installed.
+@pytest.mark.skipif(
+    "pymongo" not in sys.modules or not os.environ.get("MONGODB_HOST", False),
+    reason="Pymongo required for this test",
+)
+def test_delete_basket_mongo(test_basket):
+    """
+    Testing pantry.delete_basket(), expected to update
+    the collections in mongodb
+    """
+    file_name = "652200104150842.ch10"
+    ch10_path = str(resources.files(test_data) / file_name)
+    fs = test_basket.file_system
+    pantry_path = test_basket.pantry_path
+
+    pantry = Pantry(IndexPandas,
+                    pantry_path=pantry_path,
+                    file_system=fs)
+
+    uuid = pantry.upload_basket(
+                    upload_items=[{'path':ch10_path,
+                               'stub':False}], basket_type="test-1",
+                                metadata = {'Data Type':'ch10'}
+                               ).values.tolist()[0][0]
+
+    pantry.delete_basket(uuid)
+    mongo_client = get_mongo_db()
+    collections = ("test_supplement", "test_metadata", "test_manifest")
+    mongo_db = mongo_client["test_mongo_db"]
+    query = {'uuid': uuid}
+
+    for e in collections:
+        assert mongo_db[e].find_one(query) is None
```

Comment:
Same here, we don't want tests to upload a ch10.

Reply:
the user story mentions ch.10

Reply:
Background: Tommy wants to use the metadata database in Weave to check if a chapter 10 file has already been uploaded. However, the metadata databases are only updated manually so many developers forget to update them. Similarly, after he deletes a basket, the mongo check still says the file exists.

Reply:
I agree with @AB-USAF , no ch10 needed here.

Reply:
no more traces of ch.10 files